### ISSUE TITLE
[QUESTION] Will this repository be updated?

### ISSUE THREAD

**Issue by RaphaelRobidas** [theochem/chemtools#54]
I would love to use this tool, but I am completely unable to install it. When trying to install the master branch, I run into issues installing Horton and Libint (`#error LibInt must be compiled with an angular momentum limit of at least MAX_SHELL_TYPE`, which I was not able to fix. I also tried installing the GSoC fork (#46), but then it tries to import something from `denspart` which doesn't seem to have ever existed? (`ImportError: cannot import name 'partition' from 'denspart.mbis'`). I tried using Conda too, but with no luck.

Will there be updates to the codebase and/or documentation to allow people to use it?

For completeness
 - OS: Debian 12
 - Python version: Python 3.11.2, Miniconda with Python 3.9.16
 - NumPy version: numpy 1.24.2
 - SciPy version: scipy 1.10.1

**Comment by PaulWAyers** [comment]
@RaphaelRobidas ChemTools is under active development, so there is a lot going on in the background. Thanks for pointing out the installation difficulties. Several people have recently installed it.

[denspart](https://github.com/theochem/denspart) is a separate package, though there are partitioning pieces cluttered around the QC-Devs landscape (and we have not systematically consolidated them yet; that's been marked as a priority for 2024).

**Comment by leila-pujal** [comment]
@RaphaelRobidas , @PaulWAyers, @FarnazH, instructions to install Chemtools were posted here https://github.com/theochem/chemtools/discussions/45#discussioncomment-4077004 few months ago. We are planning to update the README.md file shortly. @RaphaelRobidas , let us know if you have any problems following these commands.

**Comment by RaphaelRobidas** [comment]
@PaulWAyers @leila-pujal  Thanks for the information, the installation instructions ended up working after a couple tricks:

- When trying to run `nosetests -v horton`, I initially got an import error regarding NumPy. It looks like NumPy stopped supporting Python 3.7 at the end of 2021, so my more recent version did not work.
- I was able to install NumPy 1.17.5 through Conda and remove my system packages from `$PYTHONPATH` (to avoid loading the recent NumPy package). With that, the Horten tests start running but end with a segmentation fault. Conda also stops working without the system packages.
- Nevertheless, I was able to install git-lfs with the system packages in `$PYTHONPATH`, then remove them to install chemtools. The chemtools tests pass and it seems to be working now.

It's a bit messy, but at least it works. Thanks!
